src/handlers/assign.rs

```diff
@@ -656,11 +656,8 @@ pub(super) async fn handle_command(
                 e.apply(&ctx.github, String::new()).await?;
                 return Ok(());
             } // we are done
-            Err(github::AssignmentError::InvalidAssignee) => {
-                issue
-                    .set_assignee(&ctx.github, &ctx.username)
-                    .await
-                    .context("self-assignment failed")?;
+            Err(AssignmentError::InvalidAssignee) => {
+                issue.set_assignee(&ctx.github, &ctx.username).await?;
```

Comment:
The `.context("self-assignment failed")?;` part could be left there, I suppose?

Reply:
We could of course call `.into()` before, but that felt unnecessary IMO.

Reply:
Yeah, it would have to be converted to `anyhow::Error` first. Nevermind.